dydx/dydxPresenters/dydxPresenters/_v4/Onboarding/Tos/dydxTosViewBuilder.swift

```diff
@@ -53,6 +56,7 @@ private class dydxTosViewPresenter: HostedViewPresenter<dydxTosViewModel>, dydxT
 
         viewModel = dydxTosViewModel()
         viewModel?.ctaAction = { [weak self] in
+            self?.onboardingAnalytics.log(step: .acknowledgeTerms)
```

Comment:
is this double-logging?

Reply:
No, but he code assign the ctaAction twice, so I will remove the redundant call to make it clear.